### PR TITLE
OSDOCS-12597: 4.16.21 RN

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.16

Issue:
[OSDOCS-12597](https://issues.redhat.com/browse/OSDOCS-12597)

Link to docs preview:
https://84836--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-21_release-notes

QE review:
N/A

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -3336,6 +3336,43 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+[source,terminal]
+----
+$ oc adm release info 4.16.21 --pullspecs
+----
+
+[id="ocp-4-16-21-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the installation program populated the `network.devices`, `template` and `workspace` fields in the `spec.template.spec.providerSpec.value` section of the {vmw-full} control plane machine set custom resource (CR). These fields should be set in the {vmw-short} failure domain, and the installation program that populated them caused unintended behaviors. Updating these fields did not trigger an update to the control plane machines, and these fields were cleared when the control plane machine set was deleted.
```
Comment (kcarmichael08) [inline release_notes/ocp-4-16-release-notes.adoc:3359]:
```suggestion
* Previously, the installation program populated the `network.devices`, `template`, and `workspace` fields in the `spec.template.spec.providerSpec.value` section of the {vmw-full} control plane machine set custom resource (CR). These fields should be set in the {vmw-short} failure domain, and the installation program that populated them caused unintended behaviors. Updating these fields did not trigger an update to the control plane machines, and these fields were cleared when the control plane machine set was deleted.
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -3336,6 +3336,43 @@ This section will continue to be updated over time to provide notes on enhanceme
...
++
+With this release, the installation program is updated to no longer populate values that are included in the failure domain configuration. If these values are not defined in a failure domain configuration, for instance on a cluster that is updated to {product-title} {product-version} from an earlier version, the values defined by the installation program are used. (link:https://issues.redhat.com/browse/OCPBUGS-44179[*OCPBUGS-44179*])
+
+* Previously, enabling ESP hardware offload using IPSec on attached interfaces in Open vSwitch broke connectivity due to a bug in Open vSwitch. With this release, OpenShift automatically disables ESP hardware offload on the Open vSwitch attached interfaces, and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-44043[*OCPBUGS-44043*])
+
+* Previously, restarting a CVO pod while it was initializing the sync work broke the guard of a blocked upgrade request. As a result, the blocked request was incorrectly accepted. with this release, the CVO postpones the reconciliation during the initialization step, and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-43964[*OCPBUGS-43964*])
```
Comment (kcarmichael08) [inline release_notes/ocp-4-16-release-notes.adoc:3365]:
```suggestion
* Previously, restarting a CVO pod while it was initializing the synchronization work broke the guard of a blocked upgrade request. As a result, the blocked request was incorrectly accepted. With this release, the CVO postpones the reconciliation during the initialization step, and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-43964[*OCPBUGS-43964*])
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 3]
```diff
@@ -3336,6 +3336,43 @@ This section will continue to be updated over time to provide notes on enhanceme
...
+
+* Previously, if you ran RHCOS in the live environment where the `rpm-ostree-fix-shadow-mode.service` used to run, the `rpm-ostree-fix-shadow-mode.service` logged a failure that did not impact the deployment or live system. With this release, the ‘rpm-ostree-fix-shadow-mode.service’ does not run when RHCOS is not running from an installed environment and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36806[*OCPBUGS-36806*])
```
Comment (kcarmichael08) [inline release_notes/ocp-4-16-release-notes.adoc:3367]:
```suggestion
* Previously, if you ran RHCOS in the live environment where the `rpm-ostree-fix-shadow-mode.service` used to run, the `rpm-ostree-fix-shadow-mode.service` logged a failure that did not impact the deployment or live system. With this release, the `rpm-ostree-fix-shadow-mode.service` does not run when RHCOS is not running from an installed environment and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-36806[*OCPBUGS-36806*])
```

**Review by kcarmichael08 — COMMENTED** [review]
Few minor comments; lgtm!

**Review by ShaunaDiaz — COMMENTED** [review]
lgtm for merge once the commits are squashed

**Pull request merged**